frontend/__mocks__/k8sResourcesMocks.ts

```diff
@@ -79,3 +79,140 @@ export const testOwnedResourceInstance: K8sResourceKind = {
     'some-filled-path': 'this is filled!',
   },
 };
+
+export const testCRDVersionV1Alpha1: CRDVersion = {
```

Comment:
I'd move this into a separate module like `frontend/__mocks__/crd.ts`. We shouldn't try to keep all mocks in a single file. It will get too large, and this change more than doubles the size.